docs/data/stashcache/run-stashcache-container.md

```diff
@@ -0,0 +1,195 @@
+Running Stashcache in a Container
+=================================
+
+The OSG operates the [StashCache data federation](/data/stashcache/overview), which
+provides organizations with a method to distribute their data in a scalable manner to thousands of jobs without needing
+to pre-stage data across sites or operate their own scalable infrastructure.
+
+[Stash Caches](/data/stashcache/install-cache) transfer data to clients such as jobs or users. A set of caches are operated across the OSG for the benefit of nearby sites;
+in addition, each site may run its own cache in order to reduce the amount of data transferred over the WAN.
+This document outlines how to run StashCache in a Docker container.
+
+Before Starting
+---------------
+
+Before starting the installation process, consider the following points:
+
+1. Docker should be running in the host you plan to install the cache
+1. User: `user` should be in the `docker` linux group.
+1. A port for XRootD to listen for incoming connections (over HTTP) (default 8000)
+1. A partition on the host to store cached data
+
+
+Running a Container
```

Comment:
I think we can get rid of this section altogether except the validation bit, which can be moved to its own validation section at the end.